api/tests/core/fraud/test_api.py

```diff
@@ -432,7 +428,7 @@ def test_age_fraud_check_ok(self, age):
             (
                 fraud_item
                 for fraud_item in result
-                if fraud_item.reason_code == fraud_models.FraudReasonCode.AGE_NOT_VALID
+                if fraud_item.reason_codes and fraud_models.FraudReasonCode.AGE_NOT_VALID in fraud_item.reason_codes
```

Comment:
Si `reason_codes` ne peut plus être `None` (comme proposé par Anthony plus haut), tu pourras simplifier en : 

```suggestion
                if fraud_models.FraudReasonCode.AGE_NOT_VALID in fraud_item.reason_codes
```

idem ailleurs